code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -570,21 +567,24 @@
 		/obj/item/twohanded/shockpaddles/syndicate/cyborg,
 		/obj/item/healthanalyzer,
 		/obj/item/surgical_drapes,
-		/obj/item/retractor,
-		/obj/item/hemostat,
-		/obj/item/cautery,
-		/obj/item/surgicaldrill,
-		/obj/item/scalpel,
-		/obj/item/melee/transforming/energy/sword/cyborg/saw,
+		/obj/item/scalpel/advanced, //scalpel+saw
+		/obj/item/retractor/advanced, //retractor+hemostat
+		/obj/item/surgicaldrill/advanced, //drill+cautery
+		/obj/item/melee/transforming/energy/sword/cyborg/saw, //we already have a tool that can perform saw steps, but c'mon, energy saws are badass (and have a high force)

```

Comment:
Can you make a `/obj/item/scalpel/advanced/cyborg` and merge that with the `/obj/item/melee/transforming/energy/sword/cyborg/saw` to reduce clutter, giving it appropriate force and sound effects if necessary?

Reply:
`/obj/item/scalpel/advanced` already has esword sounds